onnxruntime/core/providers/openvino/openvino_provider_factory.cc

```diff
@@ -233,6 +233,10 @@ static void ParseProviderInfo(const ProviderOptions& provider_options,
 
   pi.precision = OpenVINOParserUtils::ParsePrecision(provider_options, pi.device_type, "precision");
 
+  if (provider_options.contains("reshape_input")) {
+    pi.reshape = OpenVINOParserUtils::ParseInputShape(provider_options.at("reshape_input"));
```

Comment:
@preetha-intel can you please ensure check for CPU, GPU Models with the bounded inputs is added in RTM too